src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Sse41.PlatformNotSupported.cs

```diff
@@ -44,23 +44,12 @@ public abstract class Sse41 : Ssse3
         /// <summary>
         /// __m128i _mm_blendv_epi8 (__m128i a, __m128i b, __m128i mask)
         ///   PBLENDVB xmm, xmm/m128, xmm
-        /// </summary>
-        public static Vector128<sbyte> BlendVariable(Vector128<sbyte> left, Vector128<sbyte> right, Vector128<sbyte> mask) { throw new PlatformNotSupportedException(); }
```

Comment:
Right. But the diff looks like you are making them generic, instead of exploding them...

Maybe, I'm missing something?

Reply:
Ah, you are talking about SSE4.1 versions. SSE4.1 supports integer and floating both, so after exploding integer types, that will support all the 10 base types. And we have decided that should be generic.

Reply:
I believe the last API review, and the ARM Review before that, determined that we should just always explode and be explicit.

This also makes it clear when/if support for new types (like Half, Quad, etc) are added in the future that they aren't supported in the older ISAs

Reply:
CC. @terrajobst for confirmation.

Reply:
I see, but that is another story, and we have had some other generic APIs. Shall we change them at all?

Btw, I believe that Intel has different Half roadmaps from ARM.

Reply:
> Shall we change them at all

I believe so, but that is likely for a separate PR (as is moving the helper functions to the `Vector64`, `Vector128`, `Vector256` types). I pinged Immo asking for the meeting notes.

> I believe that Intel has different Half roadmaps from ARM.

Right, but I believe this was one of the "generalized" API design decisions that was made for intrinsics at the last two meetings. Which was basically: match the underlying instruction/hardware as closely as possible and always explode to avoid ambiguities from possible future expansions.

Reply:
Got it, thanks. 

> I believe so, but that is likely for a separate PR 

Let me put these changes together and open a new issue.